File: Semester 4/Programming and Projection Enironments/mpp-proiect-java/Proiect_MPP/src/main/java/root/proiect_mpp/domain/Airport.java
package root.proiect_mpp.domain;

public class Airport implements Entity<Integer> {
    private int id;
    private String name;
    private String cityName;

    // Class Constructors //

    public Airport() {
        this.id = 0;
        this.name = "";
        this.cityName = "";
    }

    public Airport(String name, String cityName) {
        this.id = 0;
        this.name = name;
        this.cityName = cityName;
    }

    public Airport(int id, String name, String cityName) {
        this.id = id;
        this.name = name;
        this.cityName = cityName;
    }

    // Getters & Setters //

    @Override
    public Integer getId() {
        return id;
    }

    @Override
    public void setId(Integer id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getCityName() {
        return cityName;
    }

    public void setCityName(String cityName) {
        this.cityName = cityName;
    }

    // toString & other functions //

    @Override
    public String toString() {
        return "Airport{" +
                "id=" + id +
                ", name='" + name + '\'' +
                ", cityName='" + cityName + '\'' +
                '}';
    }
}

File: Semester 4/Programming and Projection Enironments/mpp-proiect-csharp/Proiect_MPP/model/Airport.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace model
{
    [Serializable]
    public class Airport : Entity<int>
    {
        private string name;
        private string cityName;

        // Class Constructors //

        public Airport() : base(0)
        {
            this.name = "";
            this.cityName = "";
        }

        public Airport(string name, string cityName) : base(0)
        {
            this.name = name;
            this.cityName = cityName;
        }

        public Airport(int id, string name, string cityName) : base(id)
        {
            this.name = name;
            this.cityName = cityName;
        }

        // Getters & Setters //

        public string Name
        {
            get { return name; }
            set { name = value; }
        }

        public string CityName
        {
            get { return cityName; }
            set { cityName = value; }
        }

        // ToString & other functions

        public override string ToString() 
        {
            return "Airport{" +
                "id=" + base.ID +
                ", name='" + name + '\'' +
                '}';
        }
    }
}
